zingolib/src/lightclient.rs

```diff
@@ -463,6 +528,123 @@ impl LightClient {
         }
     }
 
+    pub async fn get_user_balances(&self) -> Result<UserBalances, LightWalletError> {
```

Comment:
What is your standard error enum type that I should use here?

Reply:
`error::ZingoLibError`